rdt/transformers/numerical.py

```diff
@@ -521,7 +521,9 @@ def _fit(self, data):
             data (pandas.Series):
                 Data to fit to.
         """
-        self._bgm_transformer = BayesianGaussianMixture(
+        sm = import_module('sklearn.mixture')
```

Comment:
do we need to assign it ? If we just did `import_module` would it work ?

Reply:
Yeah it returns the module. If I just do `import_module('sklearn.mixture')`, I get `NameError: name 'sklearn' is not defined` when trying to access the class later. Also you can't import the class directly using import_module, only modules. So you can't do `import_module('sklearn.mixture. BayesianGaussianMixture')`